pkg/clusterlinker/client.go

```diff
@@ -81,3 +85,33 @@ func (c *Client) Unlink(linkedClusterID string) error {
 
 	return nil
 }
+
+// Links returns the links of a cluster.
+func (c *Client) Links() ([]*Link, error) {
+	resp, err := c.http.Get("/cluster/v1/links")
+	if err != nil {
+		return nil, errors.New("couldn't get linked clusters")
+	}
+
+	defer resp.Body.Close()
+
+	// Common error on clusters < 1.11 and OSS clusters.
```

Comment:
Not a big fan of such comments, as they might become outdated with the time, eg. this feature is open-sourced at some point.